pages/docs/reference/whatsnew12.md

```diff
@@ -0,0 +1,460 @@
+---
+type: doc
+layout: reference
+title: "What's New in Kotlin 1.2"
+---
+
+# What's New in Kotlin 1.2
+
+## Table of Contents
+
+* [Multiplatform projects](#multiplatform-projects-experimental)
+* [Other language features](#other-language-features)
+* [Standard library](#standard-library)
+* [JVM backend](#jvm-backend)
+* [JavaScript backend](#javascript-backend)
+
+# Docs page draft: What's New in Kotlin 1.2
+
+## Multiplatform Projects (experimental)
+
+Multiplatform projects are a new **experimental** feature in Kotlin 1.2, allowing you to reuse code between target platforms supported by Kotlin – JVM, JavaScript and (in the future) Native. In a multiplatform project, you have three kinds of modules:
+
+* A *common* module contains code that is not specific to any platform, as well as declarations without implementation of platform-dependent APIs.
+* A *platform* module contains implementations of platform-dependent declarations in the common module for a specific platform, as well as other platform-dependent code.
+* A regular module that targets a specific platform and can either be a dependency of platform modules or depend on platform modules.
```

Comment:
Either use "that" in all list items or in none of them.

Reply:
Done.